src/run.jl

```diff
@@ -21,15 +21,34 @@ function replace_assignments(expr; kwargs...)
   return expr
 end
 
+function find_assignment(expr, destination)
```

Comment:
This method should get a docstring (or at least a comment on its purpose)

Reply:
Done in 99539f9.